src/packages/web-worker/utils.ts

```diff
@@ -6,10 +6,12 @@ import "./goWasmExec.d.ts";
 import { createGoWasmPvmShell } from "@/packages/web-worker/wasmGoShell.ts";
 import { logger } from "@/utils/loggerService.tsx";
 import { PvmApiInterface } from "./types.ts";
+import { createAsWasmPvmShell } from "./wasmAsShell.ts";
```

Comment:
I'd consider a full name here. At the first glace it was for me like `Create as a WASM...` but then I just had the a-ha moment. 

But I think a little bit longer `createAssemblyScriptWasmPvmShell` although it's quite long, is self-explanatory.